packages/runtime/container-runtime/src/blobManager/blobManagerSnapSum.ts

```diff
@@ -17,6 +17,7 @@ import type { ITelemetryLoggerExt } from "@fluidframework/telemetry-utils/intern
  * Information from a snapshot needed to load BlobManager
  * @legacy
  * @alpha
+ * @deprecated - This type will be moved to internal in 2.30. External usage is not necessary or supported.
```

Comment:
`@deprecated` comments don't want a leading `-`. `@param` comments are the only ones that use those.
```suggestion
 * @deprecated This type will be moved to internal in 2.30. External usage is not necessary or supported.
```

Reply:
Looks like there are a fair number of `@deprecated - ` additions that should be updated.

This may be worth a linter rule. Probably worth considering.